src/printer/embed.ts

```diff
@@ -1,5 +1,5 @@
 import { Doc, ParserOptions } from 'prettier';
-import _doc from 'prettier/doc';
+import _doc from 'prettier/doc.js';
```

Comment:
If i didn't add the `.js` it throws an error

Reply:
This is due to TS's ESM support in [4.7](https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/#esm-nodejs)

Reply:
Thank you for letting me know. I just debug that manually 😅